app/view_objects/allocations_view.rb

```diff
@@ -85,6 +93,10 @@ def requested_allocations
     @allocations.select { |allocation| allocation.request_type.in?([RequestType::INITIAL, RequestType::REPEAT]) }
   end
 
+  def confirmed_allocations
+    @allocations.select { |allocation| allocation.request_type.in?([RequestType::INITIAL, RequestType::REPEAT]) }
```

Comment:
Once we get the data finalised, this is the bit that will probably need tweaking to display the final numbers

Reply:
Why would this change? Are you thinking it should be another request type?